bpytop.py

```diff
@@ -1226,23 +1226,15 @@ class Graph:
 	colors: List[str]
 	invert: bool
 	max_value: int
+	color_max_value: int
 	offset: int
 	current: bool
 	last: int
 	symbol: Dict[float, str]
 
-	def __init__(self, width: int, height: int, color: Union[List[str], Color, None], data: List[int], invert: bool = False, max_value: int = 0, offset: int = 0):
+	def __init__(self, width: int, height: int, color: Union[List[str], Color, None], data: List[int], invert: bool = False, max_value: int = 0, offset: int = 0, color_max_value: int = None):
```

Comment:
`color_max_value: int = None`
needs to be 
`color_max_value: Union[None, int] = None`
to pass mypy type check.